02-peer-protocol.md

```diff
@@ -463,6 +466,12 @@ The sender:
     - MUST set `funding_output_contribution`
   - If it requires the receiving node to only use confirmed inputs:
     - MUST set `require_confirmed_inputs`
+  - If it wants the receiving node to contribute to the funding transaction
+    using `option_will_fund`:
+    - MUST send `request_funding` containing one of the funding rates and
+      `payment_type`s supported by the receiving node.
+  - If the previous transaction included `request_funding`:
```

Comment:
If [PR 1236](https://github.com/lightning/bolts/pull/1236) is merged, and either side can send `tx_init_rbf` then this line will need some clarification.

Only the initiator of the previous transaction should set `request_funding` for `tx_init_rbf`. The non-initiator must only set `funding_output_contribution` (as mentioned above) if they will provide an output contribution.

Same for the `The recipient` section below; `request_funding` is only expected by the non-initiator, but `funding_output_contribution` should be equal to or greater than the previous transaction that included a `request_funding`.

Note: this should also apply when the previous transaction is a splice.